guides/common/modules/ref_authentication-settings.adoc

```diff
@@ -0,0 +1,46 @@
+| *Restrict registered {SmartProxies}* | Yes | Only known {SmartProxies} can access features that use {SmartProxy} authentication.
+ifdef::satellite[]
+| *Require SSL for capsules* | Yes | Client SSL certificates are used to identify {SmartProxies} (:require_ssl should also be enabled).
+endif::[]
+| *Trusted hosts* | [] | List of hostnames, IPv4, IPv6 addresses or subnets to be trusted in addition to {SmartProxies} for access to fact/report importers and ENC output.
+| *SSL certificate* | `/etc/foreman/client_cert.pem` | SSL Certificate path that {Project} uses to communicate with its proxies.
+| *SSL CA file* | `/etc/foreman/proxy_ca.pem` | SSL CA file path that {Project} uses to communicate with its proxies.
+| *SSL private key* | `/etc/foreman/client_key.pem` | SSL Private Key path that {Project} uses to communicate with its proxies.
+| *SSL client DN env* | HTTP_SSL_CLIENT_S_DN | Environment variable containing the subject DN from a client SSL certificate.
+| *SSL client verify env* | HTTP_SSL_CLIENT_VERIFY | Environment variable containing the verification status of a client SSL certificate.
+| *SSL client cert env* | HTTP_SSL_CLIENT_CERT | Environment variable containing a client's SSL certificate.
+| *Server CA file* | | SSL CA file path used in templates to verify the connection to {Project}.
+| *Websockets SSL key* | `etc/pki/katello/private/katello-apache.key` | Private key file path that {Project} uses to encrypt websockets.
+| *Websockets SSL certificate* | `/etc/pki/katello/certs/katello-apache.crt` | Certificate path that {Project} uses to encrypt websockets.
+| *Websockets encryption* | Yes | VNC/SPICE websocket proxy console access encryption (websockets_ssl_key/cert setting required).
+| *Login delegation logout URL* | | Redirect your users to this URL on logout.
+Enable *Authorize login delegation* also.
+| *Authorize login delegation auth source user autocreate* | External | Name of the external authentication source where unknown externally authenticated users (see *Authorize login delegation*) are created.
+Empty means no autocreation.
+| *Authorize login delegation* | No | Authorize login delegation with REMOTE_USER HTTP header.
+| *Authorize login delegation API* | No | Authorize login delegation with REMOTE_USER HTTP header for API calls too.
+| *Idle timeout* | 60 | Log out idle users after the specified number of minutes.
+| *BCrypt password cost* | 9 | Cost value of bcrypt password hash function for internal auth-sources (4-30).
+A higher value is safer but verification is slower, particularly for stateless API calls and UI logins.
+A password change is needed to affect existing passwords.
+| *BMC credentials access* | Yes | Permits access to BMC interface passwords through ENC YAML output and in templates.
+| *OIDC JWKs URL* | | OpenID Connect _JSON Web Key Set_ (JWKS) URL.
+Typically \https://keycloak.example.com/auth/realms/<realm name>/protocol/openid-connect/certs when using Keycloak as an OpenID provider.
```

Comment:
Code format URL? Will the text be line-broken if we do this or will it force the table to be wider?